post-processor/googlecompute-import/post-processor.go

```diff
@@ -0,0 +1,225 @@
+package googlecomputeimport
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/storage/v1"
+
+	"github.com/hashicorp/packer/builder/googlecompute"
+	"github.com/hashicorp/packer/common"
+	"github.com/hashicorp/packer/helper/config"
+	"github.com/hashicorp/packer/helper/multistep"
+	"github.com/hashicorp/packer/packer"
+	"github.com/hashicorp/packer/post-processor/compress"
+	"github.com/hashicorp/packer/template/interpolate"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/jwt"
+)
+
+type Config struct {
+	common.PackerConfig `mapstructure:",squash"`
+
+	Bucket            string            `mapstructure:"bucket"`
+	GCSObjectName     string            `mapstructure:"gcs_object_name"`
+	ImageDescription  string            `mapstructure:"image_description"`
+	ImageFamily       string            `mapstructure:"image_family"`
+	ImageLabels       map[string]string `mapstructure:"image_labels"`
+	ImageName         string            `mapstructure:"image_name"`
+	ProjectId         string            `mapstructure:"project_id"`
+	AccountFile       string            `mapstructure:"account_file"`
+	KeepOriginalImage bool              `mapstructure:"keep_input_artifact"`
```

Comment:
`keep_input_artifact` is missing from the docs, except for that this looks good to go and that we have to sort out the tar & Go 1.10 issue.